src/Containers/KitchenCard/KitchenCard.tsx

```diff
@@ -61,7 +59,9 @@ export const KitchenCard: React.FC<KitchenProps> = ({
                 display="space-between"
                 padding="0 5px"
             >
-                <Status statusColor={status}>{status}</Status>
+                <Status statusColor={statusColor}>
+                    {status.charAt(0) + status.slice(1).toLowerCase()}
```

Comment:
Create a function for this, this will run every render, only depends on one prop